spec/jobs/pull_hotlinked_images_spec.rb

```diff
@@ -29,4 +29,42 @@
     expect(post.raw).to match(/^<img src='\/uploads/)
   end
 
+  describe 'onebox' do
+
+    Onebox::Engine::WikimediaOnebox.class_eval do
+      private
+
+      def data
+        @data ||= begin
+          {
+            link: 'https://commons.wikimedia.org/wiki/File:Brisbane_May_2013.jpg',
+            title: 'File:Brisbane May 2013.jpg',
+            image: 'http://wiki.mozilla.org/images/2/2e/Longcat1.png',
+            thumbnail: 'http://wiki.mozilla.org/images/2/2e/Longcat1.png'
+          }
+        end
+      end
+    end
+
+    let(:url) { "https://commons.wikimedia.org/wiki/File:Brisbane_May_2013.jpg" }
+
+    before do
+      SiteSetting.queue_jobs = true
```

Comment:
What is the reason for queuing the jobs in this test? Usually we enable this in tests to test that a job has been enqueued without the overheads of running the job.